pkgs/top-level/python-packages.nix

```diff
@@ -14788,6 +14788,27 @@ in modules // {
     };
   };
 
+  pyrr = buildPythonPackage rec {
+    name = "pyrr-${version}";
+    version = "0.7.0";
+
+    src = pkgs.fetchurl {
+      url = "https://pypi.python.org/packages/source/p/pyrr/pyrr-${version}.tar.gz";
+      md5 = "077454f6f9f6c690955d677d4c07985a";
+    };
+
+    # ImportError: No module named tests
+    doCheck = false;
```

Comment:
The tests are not found because they're not distributed with the source code. We generally prefer to have tests to verify whether the code is actually working. I've opened an [issue upstream](https://github.com/adamlwgriffiths/Pyrr/issues/36).